doc/tor.md

```diff
@@ -27,6 +27,11 @@ outgoing connections, but more is possible.
 	-seednode=X     SOCKS5. In Tor mode, such addresses can also be exchanged with
 	                other P2P nodes.
 
+	-onlynet=onion  Make outgoing connections only through .onion addresses. Incoming
```

Comment:
I'd say: only *to* `.onion` addresses, not through (reserve that word for the proxy), but that's a minor thing

Reply:
Thanks! Agree, will update.